<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClientDialog</class>
 <widget class="QDialog" name="ChatClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Login">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RequestUserData">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Request data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>9</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ReqRegiseter">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>35</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Received">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>301</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Disconnect">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Address">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_RoomSelected">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Rooms">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ReadyToRx">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ready to receive</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Friends">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>lineEdit_Input</tabstop>
  <tabstop>pushButton_Send</tabstop>
  <tabstop>listWidget_Rooms</tabstop>
  <tabstop>lineEdit_Address</tabstop>
  <tabstop>textEdit_RoomSelected</tabstop>
  <tabstop>pushButton_Disconnect</tabstop>
  <tabstop>pushButton_ReqRegiseter</tabstop>
  <tabstop>pushButton_Login</tabstop>
  <tabstop>pushButton_RequestUserData</tabstop>
  <tabstop>pushButton_ReadyToRx</tabstop>
  <tabstop>listWidget_Received</tabstop>
  <tabstop>pushButton_Connect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
